client/login/wp-login/login-links.jsx

```diff
@@ -64,6 +67,38 @@ export class LoginLinks extends React.Component {
 		this.props.recordTracksEvent( 'calypso_login_reset_password_link_click' );
 	};
 
+	renderBackLink() {
+		const { backTo, translate } = this.props;
+
+		// Log-ins from may include backTo, allowing them to return to the site
+		if ( backTo ) {
+			const url = urlModule.parse( backTo );
+
+			// Ensure we've got a relative URL (null) or an http[s] protocol
+			// We don't want to allow `javascript:…`
+			if ( null === url.protocol || 'http:' === url.protocol || 'https:' === url.protocol ) {
```

Comment:
Maybe `includes( [ null, 'http:', 'https:' ], protocol )`?

Reply:
I actually made a conscious decision against that when I implemented. I can't, for the life of me, remember why. Looks better 👍

Reply:
Had to roll this one back. Apparently the reason I decided was because the `includes` version doesn't work 😆 

Relative urls (`null` protocol) doesn't match for some reason 🤷‍♂️ 

Haven't investigated why that is, I can't explain it right now 😕 

```js
url.parse("/jetpack/connect")
Url {
  protocol: null,
  slashes: null,
  auth: null,
  host: null,
  port: null,
  hostname: null,
  hash: null,
  search: null,
  query: null,
  pathname: '/jetpack/connect',
  path: '/jetpack/connect',
  href: '/jetpack/connect' }
```